shared/gh/js/views/gh.admin-batch-edit.js

```diff
@@ -516,6 +516,10 @@ define(['gh.core', 'gh.constants', 'gh.utils', 'moment', 'gh.calendar', 'gh.admi
      * @private
      */
     var editableSeriesTitleSubmitted = function(value, editableField) {
+
+        // Remove the editing class from the jeditable input field
+        $('h1.editing').removeClass('editing');
```

Comment:
Can you make this more specific to the title so only one selector will ever match this?